### PR TITLE
Made list componennt 

### PR DESCRIPTION
**Pull request by poojavishnoi**
Will add code snippet later.

### CODE REVIEW

File: components/List/list.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+
+:root {
+  --list-background: rgb(245, 245, 245);
+  --list-border-color: #ff7474;
+  --list-item-color: rgb(88, 88, 88);
```
Comment (yashpatel1797) [inline components/List/list.css]: if you are following RGB then use it everywhere or use hax only try not to mix up both.

**Review by yashpatel1797 — COMMENTED** [review]
image
Add style here.

**Review by nyyshaaa — COMMENTED** [review]
looks good to me

File: components/List/list.css (hunk 1 above)
Comment (poojavishnoi) [reply components/List/list.css]: Okay

**Comment by poojavishnoi** [comment]
I forgot to import the css file.Now it's fine:)

**Review by TejasShekar — APPROVED** [review]
You haven't added code block yet. Other than that, looks good to me.

**Pull request merged**